src/main/java/no/unit/nva/model/DoiRequest.java

```diff
@@ -115,7 +115,7 @@ public Builder addMessage(DoiRequestMessage message) {
         }
 
         private void initializeMessagesIfEmpty() {
```

Comment:
Remove this, since you always initialize it in the constructor.

Reply:
empyList is immutable. I could initialize it as a new ArrayList if this is preferable.

Reply:
emptyList() creates an immutable list, so I have to create a new List for adding a message. The alternative would be to initialize the mesages list as a new ArrayList.